src/menu.js

```diff
@@ -0,0 +1,54 @@
+// Menu component
+const Menu = ({ pizzaData }) => {
+  return (
+    <main className="menu">
+      <h2>Our Menu</h2>
+
+      {/* Conditional Rendering - only render pizza list if there are pizza available (Using && operator) */}
+
+      {pizzaData.length > 0 && (
+        // {/* React Fragment - when we don't want to wrap two elements in one div */}
+        <>
+          <p>
+            Authentic italian cuisin. 6 creative dishes to choose from. All from
+            our stone own, all organic, all delicious.
+          </p>
+          <ul className="pizzas">
+            {pizzaData.map((pizza) => (
+              <Pizza pizzaObj={pizza} key={pizza.name} />
```

Comment:
why you are passing key prop from here while you are already passing whole pizza objec to Pizza component